.github/workflows/linux.yml

```diff
@@ -307,6 +262,11 @@ jobs:
         env:
           VERBOSE: 1
         run: |
+          echo "LD_LIBRARY_PATH=${LD_LIBRARY_PATH}"
```

Comment:
Is this a leftover from debugging? The lines below this one as well.

Reply:
Yes, nicely spotted. Will remove.